controllers/controller.js

```diff
@@ -50,11 +50,22 @@ const postCommentByArticleId = (req, res, next) => {
     fetchArticleById(article_id),
     insertCommentByArticleId(body, username, article_id),
   ])
-    .then((comment) => {
-      res.status(201).send({ comment: comment[2] });
+    .then((response) => {
+      const comment = { comment: response[2] };
+      res.status(201).send(comment);
     })
     .catch(next);
 };
+const patchVotesArticle = (req, res, next) => {
+  const { inc_votes: updateVotesBy } = req.body;
+  const { article_id } = req.params;
+  Promise.all([
+    fetchArticleById(article_id),
+    updateVotesCount(updateVotesBy, article_id),
+  ]).then((response) => {
```

Comment:
I know you like destructuring; you can do similar with an array when your promise all resolves:
`.then( ( [ renamedPromiseAtIndex1 ] ) => { res.status(200).send( { renamedPromiseAtIndex1 } )  } )`